views/spaces/spaces.erb

```diff
@@ -0,0 +1,8 @@
+<h1>Book a space</h1>
```

Comment:
Maybe use `<h2>` as we'll likely use `<h1>` in the header later on (e.g. for MakersBnB) but no biggie

Reply:
changed!